modules/ROOT/pages/configure-policy-template-source.adoc

```diff
@@ -4,12 +4,11 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: policy, CloudFront
 
+A policy template is a deployable JAR file that contains the business logic for how a policy must be implemented. Before a policy is applied to an API, the policy template is downloaded and accessed to determine the policy dependencies.
```

Comment:
The statement here is confusing. "A policy template is a deployable JAR file that contains the business logic for how a policy must be implemented." -> "A policy template is a deployable JAR file that contains its business logic."

Reply:
The statement is incorrect. "Before a policy is applied to an API, the policy template is downloaded and accessed to determine the policy dependencies."
You may remove it completely, or explain what's the difference among JAR, YAML and pom files.

Reply:
This topic is about the policy template and this sentence is explaining what happens to it once you specify the source. I think we should keep it here. We are not talking about JAR and YAML files as they are irrelevant to this topic. They should be covered in the topic where we discuss policies.